app/models/invenio_rdm_record_converter.rb

```diff
@@ -12,19 +12,23 @@ class InvenioRdmRecordConverter < Sufia::Export::Converter
   ENGLISH = "english"
   CREATOR_UNKNOWN = 'unknown'
   CREATOR_NOT_IDENTIFIED = "creator not identified."
-  MEMOIZED_PERSON_OR_ORG_DATA_FILE = 'memoized_person_or_org_data.txt'
   ROLE_OTHER = 'other'
+  DEFAULT_RIGHTS_SCHEME = "spdx"
```

Comment:
Some very small refactoring here. I added a constant for one of the file paths and also moved the class variables from the initializer function to the class level. Moving the class variables out of the initializer function makes it easier to do unit testing since we no longer have to create an instance of the class to have these variables populate.